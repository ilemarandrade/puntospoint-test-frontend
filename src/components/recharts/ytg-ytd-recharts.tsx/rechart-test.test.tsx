import React from 'react';
import { render } from '@testing-library/react';
import Rechart from '.';
import ytgydtDataMock from '@/datamock/ytg-ytd-mock';

describe('Rechart Component', () => {
  it('renders correctly with Clients tag', async () => {
    const { container } = render(
      <div className="w-[80vw]">
        <Rechart data={ytgydtDataMock} />
      </div>
    );
    expect(container).toMatchSnapshot();
  });
});
