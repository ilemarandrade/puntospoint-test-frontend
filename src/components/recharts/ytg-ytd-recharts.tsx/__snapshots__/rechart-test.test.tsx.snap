// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rechart Component renders correctly with Clients tag 1`] = `
<div>
  <div
    class="w-[80vw]"
  >
    <div
      class="MuiGrid2-root MuiGrid2-container MuiGrid2-direction-xs-row relative css-1jxwjqx-MuiGrid2-root"
    >
      <div
        class=" "
      />
      <div
        class="MuiGrid2-root MuiGrid2-direction-xs-row MuiGrid2-grid-xs-12 MuiGrid2-grid-sm-6 css-1lu792j-MuiGrid2-root"
      >
        <div
          class="recharts-responsive-container"
          style="width: 100%; height: 100%; min-width: 200px; min-height: 500px;"
        />
      </div>
      <div
        class="MuiGrid2-root MuiGrid2-direction-xs-row MuiGrid2-grid-xs-12 MuiGrid2-grid-sm-6 css-1lu792j-MuiGrid2-root"
      >
        <div
          class="recharts-responsive-container"
          style="width: 100%; height: 100%; min-width: 200px; min-height: 500px;"
        />
      </div>
    </div>
  </div>
</div>
`;
